resources/views/api/emails/voucher_duplicate_email_textonly.blade.php

```diff
@@ -0,0 +1,22 @@
+* Voucher Duplicate Entered *
+
+==============================================
+
+[xXx]
+Hi {{ config('mail.to_admin.name') }},
+
+{{ $user }} has tried to submit voucher
+{{ $vouchercode }} against
+{{ $trader }} of
+{{ $market->name ?? 'no associated market' }}'s account, however that voucher has already been submitted by another trader.
```

Comment:
It doesn't work if you put it in there, because then the blade template is trying to get the name of something that isn't an object and it barfs. This is the only way I could get it to work?